### DISCUSSION TITLE
How can I edit the filter dialog ?

### DISCUSSION THREAD

**Discussion by togisoft** [palexdev/MaterialFX#241] [Archived]
How can I change the filter dialog color?. When the scrollbar is active, the filter does not open in the center of the screen. Is there a solution to this?

![filter](https://user-images.githubusercontent.com/60444557/187224029-5ee0d453-6208-4b81-aabf-02affd0e2e0a.PNG)

**Comment by stefanofornari** [comment]
It's a bit tricky, see answer #255 and for a working example https://github.com/stefanofornari/EasyWallet/blob/13f5d01c601f37741065766188f624dd2e339f1f/src/main/java/ste/w3/easywallet/ui/LedgerDialog.java (just as starting point)

HTH
